File: app/src/main/java/com/icerockdev/babenko/ui/images/ImagesInteractor.java
package com.icerockdev.babenko.ui.images;

import com.icerockdev.babenko.model.entities.ImageItem;

import java.util.List;

import io.reactivex.Single;

/**
 * Created by Roman Babenko on 10/05/17.
 */

public interface ImagesInteractor {
    Single<List<ImageItem>> requestPicturesList();
}

File: app/src/main/java/com/icerockdev/babenko/ui/images/ImagesInteractor.kt
package com.icerockdev.babenko.ui.images

import com.icerockdev.babenko.model.entities.ImageItem

import io.reactivex.Single

/**
 * Created by Roman Babenko on 10/05/17.
 */

interface ImagesInteractor {
    fun requestPicturesList(): Single<List<ImageItem>>
}
